projects/telemetry/src/translate.c

```diff
@@ -0,0 +1,39 @@
+#include "translate.h"
+#include <inttypes.h>
+#include <stdio.h>
+#include <string.h>
+
+static StatusCode prv_can_rx(const CANMessage *msg, void *context, CANAckStatus *ack_reply) {
+  // Note that this currently responds to ACKs
```

Comment:
How do we want to determine if stuff responds to an ACK? I feel like only the intended receiver should respond but I'm not sure how best to enforce that other than in code reviews?

Reply:
To be clear, this should be "responds with ACKs to critical messages". I don't think there's any value in trying to make things not respond. Instead, we should be checking device IDs in our ACK handlers.

Reply:
I think the bitset instead of num_expected_acks is a really good idea. :D

Reply:
👍 

Tracking with ELEC-280